sentences/fr/_common.yaml

```diff
@@ -524,19 +524,21 @@ expansion_rules:
   partout: ([de] partout)|(<dans> [<tous>] [<le>]<maison>)|(<dans> toutes les pièces)
 
   # Verbs
-  regle: "(règle|régler|met|mets|mettre|ajuste|ajuster|change|changer)"
+  active: "(active|activer|joue|jouer|exécute|exécuter)"
```

Comment:
Don;t you find it weird that `active` contains `exécute` and that `execute` is also an expansion rules containing other verbs 🤔 ?

I wonder if we really need to split these two in fact.
The main difference between a scene and a script is that a script is action-based (Do this) and a scene is state-based (Put that device in that state)

But when it comes to triggering them by voice, I wonder if you cannot have a single verb list.

- Joue le script suivant.
- Joue a scène suivante.

It does not bother me to be honest.

(English as the same verb list too ;) )

Reply:
You're right: the difference between scripts and scenes is subtle. I merge in one unique expansion rule (active).

However, am I correct in this approach? A verb expansion rule is a verb + all synonyms used in a specific context or with specific domains.

It's quite tricky to find the right balance without over-complicating things :)

Reply:
I would say an expansion rule should be verbs + synonyms without a particular context.

But I agree with you, the balance is quite complex.
For example, most of the tokens in the expansion rule `<allume>` are now part of `<active>`

I think this is it: A balance.
If we want to be perfectly correct, always, then we should not use expansion rules, and define what verb we want to use in every sentence, but that would make the whole repo too complex to read.

I think today the balance is OK, we do have some weird sentences because of our expansion rule, for example, you can say `démarre la lumière`... That one could be worth adapting a bit our expansion rules.

I do like the idea of having expansion rules per domains... 
I'll think about it cc @piitaya